docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
     build: ./frontend
     volumes:
       - './frontend:/app'
-      - 'node_modules:/app/node_modules'
+      - '/app/node_modules'
```

Comment:
I went back to anon volumes. You can do `docker-compose up --build -V` to nuke the built node_modules when deps update.

Reply:
Worked perfectly fine, thanks for the hint!